api/controller/remoterelations/remoterelations.go

```diff
@@ -160,16 +160,10 @@ func (c *Client) RemoteApplications(ctx context.Context, applications []string)
 // WatchRemoteApplications returns a strings watcher that notifies of the addition,
 // removal, and lifecycle changes of remote applications in the model.
 func (c *Client) WatchRemoteApplications(ctx context.Context) (watcher.StringsWatcher, error) {
-	var result params.StringsWatchResult
-	err := c.facade.FacadeCall(ctx, "WatchRemoteApplications", nil, &result)
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	if result.Error != nil {
-		return nil, result.Error
-	}
-	w := apiwatcher.NewStringsWatcher(c.facade.RawAPICaller(), result)
-	return w, nil
+	// todo(gfouillet): re-enable this watcher call whenever CMR will be fully
+	//   implemented. It is required to disable that way because this watcher
```

Comment:
todo: s/implemented./implemented in the new domain./

For all. A tiny bit more explicit.